lib/db/transactions.go

```diff
@@ -714,7 +713,7 @@ func (t readWriteTransaction) updateGlobal(gk, keyBuf, folder, device []byte, fi
 			// The inserted file is the global file
 			global = file
 		} else {
-			keyBuf, global, err = t.getGlobalFromFileVersion(keyBuf, folder, name, true, globalFV)
+			keyBuf, global, _ = t.getGlobalFromFileVersion(keyBuf, folder, name, true, globalFV)
```

Comment:
I suspect that we should in fact have some sort of error handling here, so the bug is larger than just unused retval

Reply:
Generally I want to have a good look at all this db changes, maybe there's some insights (bugs) to be discovered.

Reply:
On a second, slower look it all looks entirely benign. Just as noted here where a `if err != nil { return ..., err}` has been forgotten previously.

Reply:
Inserted an error check.